docs/migration-guides/mathjs/benchmark/benchmark.exp.number.js

```diff
@@ -62,7 +62,30 @@ bench( pkg+'::stdlib:math/base/special/exp:value=number', opts, function benchma
 	b.end();
 });
 
-// TODO: add math/special/exp benchmarks
+// Add math/special/exp benchmarks
+bench( pkg+'::stdlib:math/special/exp:value=number', opts, function benchmark( b ) {
+    var x;
+    var y;
+    var i;
+
+    x = filledBy( 100, uniform( -100.0, 100.0 ) );
+
+    b.tic();
+    for ( i = 0; i < b.iterations; i++ ) {
+        y = exp( x[ i%x.length ] );
```

Comment:
This package does not exist. The TODO was a placeholder to revisit these benchmarks once `@stdlib/math/special/exp` was added to the project.